app/javascript/angular/code/controllers/_sessions_list_ctrl.js

```diff
@@ -26,28 +26,23 @@ export const SessionsListCtrl = (
     $window.sessions = sessions = $scope.sessions;
     $scope.sessionsForList = [];
 
-    // prolly this can be removed
-    if (_(params.get("selectedSessionIds", [])).isEmpty()) {
-      params.update({ selectedSessionIds: [] });
-    }
-
     sessions.reSelectAllSessions();
   };
 
   $scope.isSessionDisabled = function(sessionId) {
     // disabled if there is another selected session and it is not the selected session
     // when refactoring to a radio button this should always be false
     return (
-      !params.get("selectedSessionIds", []).includes(sessionId) &&
-      sessions.hasSelectedSessions()
+      !(sessionsUtils.selectedSessionId() === sessionId) &&
+      sessionsUtils.isSessionSelected()
     );
   };
 
   $scope.$watch(
     "params.get('map')",
     ({ hasChangedProgrammatically }) => {
       console.log("watch - params.get('map')");
-      if (sessions.hasSelectedSessions()) return;
```

Comment:
The fact that this was working before is MAGIC! hasSelectedSessions was calling a chain of at least 5 other methods and at the end is a function that does

  find: function(self, id) {
    return _(self.sessions || []).detect(function(session) {
      return session.id === id;
    });
  },
But when a page with selected session is first loaded (or refreshed) the self.sessions is empty so sessions.hasSelectedSessions() returned false event though session was selected 😮

Reply:
The magic of js